styles.css

```diff
@@ -143,6 +147,19 @@ img {
   border-radius: 32px;
 }
 
+.next img {
+  margin: 0;
+  padding: 0;
+}
+
+.contact img {
+  height: 12.88px;
+  padding: 20px;
+  width: 10px;
```

Comment:
It is recommended to use rem or % units to create a responsive design.

Reply:
Noted Thanks!